### ISSUE TITLE
[2.11.0 regression] Unexpected exception: gameViewPtr is not initialized

### ISSUE THREAD

**Issue by FabioPedretti** [lincity-ng/lincity-ng#147]
2.11.0 built on Ubuntu 24.04 crash while starting a game.

Note that:
- 2.10.2 was fine
- the issue happens with all the maps with a "dice" (the first 4, from river delta to swap), while the others (from Beach to good_times) start fine

How to reproduce:
- Cleaned old config with `rm -rf ./.local/share/lincity-ng`
- Run the game -> New Game -> river delta -> Start
- The game crash with:
```
$ lincity-ng 
Starting lincity-ng (version 2.11.0-unknown)...
[/home/user/.local/share/lincity-ng/] is in the search path.
[/usr/share/lincity-ng] is in the search path.
[/home/user/.local/share/lincity-ng/] is the write directory.
Language is "it_IT".
 fast = 10

OpenGL Mode 1024x768
File found: 'music/default/default.xml'. Loading song data...
Found song: 'Robert van Herk - City Blues'
Found song: 'Robert van Herk - Architectural Contemplations'
Found song: 'Pronobozo - Lincity'
world id: 710691444
carving river ... done
setting up ecology ... done
Unexpected exception: gameViewPtr is not initialized
```

**Comment by dbear496** [comment]
Thanks for catching this. I was able to reproduce the bug, so it shouldn't be too difficult to fix.

**Comment by dbear496** [comment]
In the mean time, a workaround is to first start a new game with one of the non-dice scenarios (e.g. Beach), and then immediately go back to the menu and choose the desired dice scenario.

**Comment by dbear496** [comment]
I implemented a fix in PR #149 and am waiting for review (in case anyone cares to review). If you don't want to wait, you can get new fixes as I create them by tracking master on [my fork](https://github.com/dbear496/lincity-ng).
